releases.json

```diff
@@ -223,6 +223,17 @@
       "0.4.1-1"
     ]
   },
+  "avro-cpp": {
+    "dependency_names": [
+      "avro-cpp"
+    ],
+    "program_names": [
+      "avrogencpp"
+    ],
+    "versions": [
+      "1.12.99-1"
```

Comment:
I think if you just include the hash here instead of the version you can work around the sanity check issue. So `82a2bc8b034de34626e2ab8bf091234122474d50-1`

Reply:
I'm not quite sure what exactly will do the trick, since differences between the json and the meson build file will lead to some error at some point. Do you happen to know when there will be a new avro release? That might be the easiest...

Reply:
BTW, iceberg-cpp is also depending on a hash tag of avro-cpp: https://github.com/apache/iceberg-cpp/blob/main/cmake_modules/IcebergThirdpartyToolchain.cmake#L186